source/lib/JMSLab/autofill.py

```diff
@@ -32,6 +32,15 @@ def GenerateAutofillMacros(autofill_lists,  autofill_formats, autofill_outfile):
     .tex file
 
     """
+    if type(autofill_lists) != list:
+            raise Exception("Argument 'autofill_lists' must be list")
```

Comment:
Just asking: does this stop the execution of the program?

Reply:
@santiagohermo yes indeed it does! Do we want to raise a warning instead?